charts/sn-platform/templates/openshift/_openshift.tpl

```diff
@@ -0,0 +1,3 @@
+{{- define "pulsar.openshift.scc.name" -}}
+{{ template "pulsar.fullname" . }}-openshift-scc
```

Comment:
Maybe it's better to add the namespace to the name to reduce potential conflicts, but it might be ok for now.

Reply:
Make sense to me. Updated